docs/static/releasenotes.asciidoc

```diff
@@ -3,81 +3,46 @@
 
 This section summarizes the changes in the following releases:
 
-* <<logstash-6-0-0-rc2,Logstash 6.0.0-rc2>>
-* <<logstash-6-0-0-rc1,Logstash 6.0.0-rc1>>
-* <<logstash-6-0-0-beta2,Logstash 6.0.0-beta2>>
-* <<logstash-6-0-0-beta1,Logstash 6.0.0-beta1>>
-* <<logstash-6-0-0-alpha2,Logstash 6.0.0-alpha2>>
-* <<logstash-6-0-0-alpha1,Logstash 6.0.0-alpha1>>
+* <<logstash-6-0-0,Logstash 6.0.0>>
 
 ifdef::include-xpack[]
 See also:
 
 * <<release-notes-xls>>
 endif::include-xpack[]
 
-[[logstash-6-0-0-rc2]]
-=== Logstash 6.0.0-rc2 Release Notes
+[[logstash-6-0-0]]
+=== Logstash 6.0.0 Release Notes
 
+* Breaking: The setting `config.reload.interval` has been changed to use time value strings such as `5m`, `10s`, etc for
+  convenience. Previously, users had to convert this to a second time value themselves.
+* Breaking: The list of plugins bundled with Logstash 6.0.0-beta1 release has changed. Please consult the breaking changes document
+  for a complete list.
+* Breaking: Currently, when Logstash is installed and set up via package managers, it loads all files found
+  in `/etc/logstash/conf.d` as configuration. This can be problematic if there are non-configuration files in this
+  folder. Starting from alpha2, Logstash will only glob files ending with `.conf` extension in `/etc/logstash/conf.d` ({lsissue}6979)[Issue 6979)]).
+* Added new `logstash.yml` setting: `config.support_escapes`. When enabled, Logstash will interpret escape sequences in
+  strings in the pipeline configuration.
+* Added an explicit `--setup` phase for modules which is used to install dashboards and Elasticsearch template.
```

Comment:
You have a dangling modifier here. Change to:

Added an explicit `--setup` phase for modules. This phase is used to install dashboards and the Elasticsearch template.